### PR TITLE
feat: support  podwise.ai

### PR DESCRIPTION
**Pull request by lyling**
## Example for the Proposed Route(s) / 路由地址示例

```routes
/podwise/explore
/podwise/explore/latest
/podwise/explore/all
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

当网站时间为的月份缩写(MMM)与浏览器地区不一致时， 需要指定parseDate的语言参数，如'en', 如果发现parseDate为Invalid Date时, 调用

```js
import localeData from 'dayjs/plugin/localeDate';
dayjs.extent(localeData); 
dayjs.monthShort();
```
来查看缩写是否与时间的月份一致

### CODE REVIEW

File: lib/routes/podwise/episodes.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            list.map((item) =>
+                cache.tryGet(item.link, async () => {
+                    const response = await ofetch(item.link);
+                    const $ = load(response);
+
+                    item.description = $('summary').first().html();
+                    return item;
+                })
+            )
+        );
```
Comment (TonyRL) [inline lib/routes/podwise/episodes.ts:68]: Consider adding the audio url and make it becomes a [podcast feed](https://docs.rsshub.app/joinus/advanced/advanced-feed#create-a-podcast-feed).

File: lib/routes/podwise/episodes.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+        return {
+            title,
+            description,
+            item: items,
+        };
```
Comment (TonyRL) [inline lib/routes/podwise/episodes.ts:74]: Missing `link`.

File: lib/routes/podwise/namespace.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    url: 'podwise.ai',
+    description: `
+:::tip
+Podwise provides  none of official RSS feeds.
+This route support below:
+
+-   Collections: \`https://podwise.ai/explore\`
+-   Episodes: \`https://podwise.ai/explore/:category\`
+-   Episode: \`https://podwise.ai/dashboard/episodes/:episode\`
+:::`,
```
Comment (TonyRL) [inline lib/routes/podwise/namespace.ts:14]: No need to explictly mention that the site does not provide official RSS feeds.

File: lib/routes/podwise/collections.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    example: '/podwise/explore',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
```
[the remark this answers is not in the document]
Comment (lyling) [reply lib/routes/podwise/collections.ts]: done

File: lib/routes/podwise/collections.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    radar: [
+        {
+            source: ['podwise.ai', 'podwise.ai/explore'],
+        },
+    ],
+    name: 'Collections',
+    maintainers: ['lyling'],
+    handler: async () => {
+        const link = `https://podwise.ai/explore`;
```
[the remark this answers is not in the document]
Comment (lyling) [reply lib/routes/podwise/collections.ts]: done

File: lib/routes/podwise/collections.ts [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+                    link,
+                    description,
+                };
+            });
+
+        const items = list.map((item) => ({
+            title: item.title,
+            link: `https://podwise.ai${item.link}`,
+            description: item.description,
+        }));
```
[the remark this answers is not in the document]
Comment (lyling) [reply lib/routes/podwise/collections.ts]: done

File: lib/routes/podwise/collections.ts [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+        return {
+            title: $('title').text(),
+            description: collectinDescription,
+            item: items,
+        };
```
[the remark this answers is not in the document]
Comment (lyling) [reply lib/routes/podwise/collections.ts]: done

File: lib/routes/podwise/collections.ts [hunk 8]
```diff
@@ -0,0 +1,60 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+
+export const route: Route = {
+    path: '/explore',
+    categories: ['multimedia', 'programming'],
```
[the remark this answers is not in the document]
Comment (lyling) [reply lib/routes/podwise/collections.ts]: done

File: lib/routes/podwise/episodes.ts [hunk 9]
```diff
@@ -0,0 +1,76 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/explore/:type',
+    categories: ['multimedia', 'programming'],
```
[the remark this answers is not in the document]
Comment (lyling) [reply lib/routes/podwise/episodes.ts]: done

File: lib/routes/podwise/episodes.ts [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+    example: '/podwise/explore/latest',
+    parameters: { type: 'latest or all episodes.' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
```
[the remark this answers is not in the document]
Comment (lyling) [reply lib/routes/podwise/episodes.ts]: done

File: lib/routes/podwise/episodes.ts (hunk 1 above)
Comment (lyling) [reply lib/routes/podwise/episodes.ts:60]: actally it is not the original podcast. i can`t get podcast link from html.

File: lib/routes/podwise/episodes.ts (hunk 2 above)
Comment (lyling) [reply lib/routes/podwise/episodes.ts]: done

File: lib/routes/podwise/namespace.ts (hunk 3 above)
Comment (lyling) [reply lib/routes/podwise/namespace.ts]: done

File: lib/routes/podwise/episodes.ts (hunk 1 above)
Comment (TonyRL) [reply lib/routes/podwise/episodes.ts:60]: You can extract podcast data using regular expression.
```ts
                    const nextData = JSON.parse(
                        $('script:contains("podName")')
                            .first()
                            .text()
                            .match(/self\.__next_f\.push\((.+)\)/)?.[1] ?? ''
                    );
                    const podcastData = JSON.parse(nextData[1].slice(2))[1][3].children[3].episode;
```

Comment (lyling) [reply lib/routes/podwise/episodes.ts]: done

File: lib/routes/podwise/episodes.ts [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+                    );
+                    const podcastData = JSON.parse(nextData[1].slice(2))[1][3].children[3].episode;
+
+                    // rss feed
+                    item.title = podcastData.title;
+                    item.author = podcastData.podName;
+
+                    // podcast feed
+                    item.itunes_item_image = podcastData.cover;
+                    item.itunes_duration = parseDuration($duration.text());
```
Comment (TonyRL) [inline lib/routes/podwise/episodes.ts:75]: You've got `sampleDuration` which already is the full duration 
https://github.com/DIYgod/RSSHub/pull/15670#discussion_r1613641283

File: lib/routes/podwise/episodes.ts [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+                    item.enclosure_url = podcastData.link;
+                    // item.enclosure_length: nothing can convert to.
+                    item.enclosure_type = podcastData.linkType;
```
Comment (TonyRL) [inline lib/routes/podwise/episodes.ts:78]: This is the MIME type of the file. Please use `audio/mpeg` for mp3 and `audio/x-m4a` for m4a

File: lib/routes/podwise/episodes.ts (hunk 11 above)
Comment (lyling) [reply lib/routes/podwise/episodes.ts:75]: i have checked the sample duration . it is not equal to the value from the page.

File: lib/routes/podwise/episodes.ts (hunk 12 above)
Comment (lyling) [reply lib/routes/podwise/episodes.ts]: done

File: lib/routes/podwise/collections.ts [hunk 13]
```diff
@@ -0,0 +1,46 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+
+export const route: Route = {
+    path: '/explore',
+    categories: ['multimedia'],
+    example: '/podwise/explore',
+    radar: [
```
Comment (jonesjoebloggsmorganwalker) [inline lib/routes/podwise/collections.ts:9]: Personal
Checking
Savings & CDs
Credit Cards
Home Loans
Personal Loans
Auto Loans
Premier````

````
Education & Tools
Investing & Wealth Management
Small Business
Commercial Banking
Corporate & Investment Banking
Good afternoon
Sign on to manage your accounts.

Username

Password
 Save username
 Enroll
Forgot username or password?
Security Center
Privacy, Cookies, and Legal

$300 checking bonus on us
New customers open an eligible checking account with qualifying electronic deposits
Get started >>
 
Want a $125 checking bonus?
Open a new Clear Access Banking account and complete the qualifying requirements

Learn more
 
Earn 20,000 bonus points
Plus, earn 3X points on many everyday categories with $0 annual fee. Terms apply.

Learn more
 
Open a savings account
Explore our savings accounts and find the right fit for you

Get started
Interest rates today


Check rates
 
Earn up to 60,000 bonus points
on qualifying purchases. Terms apply.

Learn more
‍
Financial guidance and support

Get tools. Get tips. Get peace of mind.

Discover digital tools to help you budget, save, manage credit, and more
Access the toolkit


Reimagine. Navigate. Move forward.

Discover steps that may help you manage life-changing events
Manage life events


Save. Invest. Retire well.

Discover how to start saving to meet your retirement goals
Get ready for retirement


Banking in the palm of your hand
Our Wells Fargo Mobile® app gives you fast and secure access to your finances
Check your account balance
View your latest FICO® Score1
Send and receive money with Zelle®2

Comment (jonesjoebloggsmorganwalker) [inline lib/routes/podwise/collections.ts:9]: Personal
Checking
Savings & CDs
Credit Cards
Home Loans
Personal Loans
Auto Loans
Premier````

````
Education & Tools
Investing & Wealth Management
Small Business
Commercial Banking
Corporate & Investment Banking
Good afternoon
Sign on to manage your accounts.

Username

Password
 Save username
 Enroll
Forgot username or password?
Security Center
Privacy, Cookies, and Legal

$300 checking bonus on us
New customers open an eligible checking account with qualifying electronic deposits
Get started >>
 
Want a $125 checking bonus?
Open a new Clear Access Banking account and complete the qualifying requirements

Learn more
 
Earn 20,000 bonus points
Plus, earn 3X points on many everyday categories with $0 annual fee. Terms apply.

Learn more
 
Open a savings account
Explore our savings accounts and find the right fit for you

Get started
Interest rates today


Check rates
 
Earn up to 60,000 bonus points
on qualifying purchases. Terms apply.

Learn more
‍
Financial guidance and support

Get tools. Get tips. Get peace of mind.

Discover digital tools to help you budget, save, manage credit, and more
Access the toolkit


Reimagine. Navigate. Move forward.

Discover steps that may help you manage life-changing events
Manage life events


Save. Invest. Retire well.

Discover how to start saving to meet your retirement goals
Get ready for retirement


Banking in the palm of your hand
Our Wells Fargo Mobile® app gives you fast and secure access to your finances
Check your account balance
View your latest FICO® Score1
Send and receive money with Zelle®2

**Pull request merged**